locale/fr/redmew.cfg

```diff
@@ -0,0 +1,41 @@
+[retailer]
+market_name=Maché
+no_items_in_market=Pas d'item disponible actuellement
+item_sold_out=EPUISÉ !
+item_is_free=GRATUIT !
+generic_item_disabled_message=Item desactivé
+not_enough_currency=Il manque __1__ __2__ pour acheter __3__
+item_with_player_limit_description=You have bought this item __1__ out of __plural_for_parameter_2_{1=1 time|rest=__2__ times}__
+item_no_longer_available=Cet item n'est plus disponible dans le marché
+item_disabled_reason=__1__est desactivé. __2__
+no_inventory_space=Taille de l'inventaire insuffisant
+
+[common]
+fail_no_target=Pas de joueur trouvé avec le nom : __1__
+warn_no_target=Attention : le joueur  __1__ n'as pas été trouvé, mais la commandé a été executé.
+close_button=Fermer
+server_unavailable=Le serveur est actuellement indisponible.
+player_name=Nom du joueur
+connection_status=Status de connection
+online=En ligne
+offline=Hors ligne
+player_index=Index des joueurs
+player_rank=Rang des joueurs
+yes=oui
+no=non
+time_played=Temps joué
+afk_time=Temps d'inactivité
+current_force=Current force
+current_surface=Current surface
```

Comment:
Can i have the context ?

Reply:
Think the context was removed, these keys are never used.
In fact only these three are used from common: "available" "quantity" "fail_no_target"